boards/FSM/Src/main.c

```diff
@@ -176,6 +183,9 @@ int main(void)
 
     Io_WheelSpeedSensors_Init(&htim16, &htim17);
 
```

Comment:
Wondering if this is an acceptable way to intialize the adc. I couldn't think of a nice way to call Io_SharedAdc_Create inside Io_SteeringAngleSensor because every time we run Io_SharedAdc_Create we are creating a new ADC. It's quite different compared to intializing a timer channel for a wheel speed sensor.